packages/cli/src/util/__tests__/PackageManager-test.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+import ChildProcess from 'child_process';
+import PackageManager from '../PackageManager';
+import yarn from '../yarn';
+
+const PROJECT_DIR = '/project/directory';
+const PACKAGES = ['react', 'react-native'];
+const PACKAGE = '@babel/core';
+const EXEC_OPTS = { stdio: 'inherit' };
+
+beforeEach(() => {
+  jest.spyOn(ChildProcess, 'execSync').mockImplementation(() => {});
+});
+
+describe('yarn', () => {
+  beforeEach(() => {
+    jest
+      .spyOn(yarn, 'getYarnVersionIfAvailable')
+      .mockImplementation(() => true);
+    jest.spyOn(yarn, 'isGlobalCliUsingYarn').mockImplementation(() => true);
+  });
+
+  it('should install', () => {
+    const uut = new PackageManager({ projectDir: PROJECT_DIR });
```

Comment:
`uut`?

Reply:
Unit Under Test. Can change that if it's confusing :P